numpy/core/src/umath/override.c

```diff
@@ -555,26 +551,32 @@ PyUFunc_CheckOverride(PyUFuncObject *ufunc, char *method,
             npy_cache_import("numpy.core._internal",
                              "array_ufunc_errmsg_formatter",
                              &errmsg_formatter);
+
             if (errmsg_formatter != NULL) {
+                /* All tuple items must be set before use */
+                Py_INCREF(Py_None);
+                PyTuple_SET_ITEM(override_args, 0, Py_None);
                 errmsg = PyObject_Call(errmsg_formatter, override_args,
                                        normal_kwds);
                 if (errmsg != NULL) {
                     PyErr_SetObject(PyExc_TypeError, errmsg);
                     Py_DECREF(errmsg);
                 }
             }
+            Py_DECREF(override_args);
             goto fail;
         }
 
         /*
          * Set the self argument of our unbound method.
          * This also steals the reference, so no need to DECREF after.
          */
-        PyTuple_SetItem(override_args, 0, override_obj);
+        PyTuple_SetItem(override_args, 0, override_obj)
```

Comment:
The `;` was removed incorrectly! Also, looking at this again, this can now really be a `SET_ITEM`.